packages/stark-ui/src/modules/pretty-print/components/pretty-print.component.ts

```diff
@@ -151,12 +155,19 @@ export class StarkPrettyPrintComponent extends AbstractStarkUiComponent implemen
 			try {
 				switch (this.format) {
 					case "xml":
+						prismGrammar = Prism.languages.markup;
+						prismClass = prismClassPrefix + "markup";
+						this.prettyString = prettier.format(this.data, {
+							parser: "xml",
+							plugins: prettierPlugins,
+							xmlWhitespaceSensitivity: "ignore"
+						});
+						break;
+
 					case "html":
 						prismGrammar = Prism.languages.markup;
 						prismClass = prismClassPrefix + "markup";
-						// FIXME Adapt following line after replacing "pretty-data" dependency. See: https://github.com/NationalBankBelgium/stark/issues/2543
-						// this.prettyString = prettyData.xml(this.data);
-						this.prettyString = this.data;
+						this.prettyString = prettier.format(this.data, { parser: "angular", plugins: prettierPlugins });
```

Comment:
Just one thing, maybe it's worth mentioning in the changelog, with this change now the PrettyPrint component is able to format not only HTML but also Angular HTML code.

So what to do? 🤔

Reply:
Indeed, I extended the support to Angular.
I think it could be a good idea since it's an angular framework 😊
Or maybe should it be a separated feature?

It would mean changing back to html and add a new support for "angular"?

Reply:
Ah ok, well, for me it's ok to keep it, but maybe just mention this new feature in the changelog?